src/widgets/helper/color/SBCanvas.hpp

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <QWidget>
+
+namespace chatterino {
+
+class SBCanvas : public QWidget
```

Comment:
Comment on the class what it is since "SB" isn't a known acronym to me, or just rename it to its full name